roles/update-host/handlers/main.yml

```diff
@@ -0,0 +1,4 @@
+---
+
+- name: reboot the host
+  reboot:
```

Comment:
Suggestion: Please add the parameters as "optional" to allow users to specify things like `reboot_timeout`, etc.